_posts/2019-10-30-upgrade-reminder.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: Make sure you are ready for the Bitcoin Cash network upgrade
+subtitle: Upgrade activates Nov. 15th
+multiLangId: 2019-10-30-upgrade-reminder
+lang: en
+---
+
+## What is happening?
+
+In just over two weeks, on Nov 15th 2019, the Bitcoin Cash network will undergo a planned network upgrade. This upgrade is a routine scheduled process, as part of the [Bitcoin Cash Roadmap](https://www.bitcoincash.org/roadmap.html).
+
+## What do I need to do?
+
+All node operators will need to update their node software to a recent version. For Bitcoin ABC users, this means any version 0.20.0 or above, which can be downloaded [here](https://download.bitcoinabc.org/).
+
+Anyone running a service such as a block explorer, and exchange, or other services provided by a full node needs to upgrade.
```

Comment:
`and exchange` -> `an exchange`

Reply:
`needs` -> `need`

Reply:
Ok, I did something slightly different with need/needs. Since it goes with "anyone" it should be needs, but the other clause in the sentence also should be singular.